Android.mk

```diff
@@ -0,0 +1,26 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := NominatimNlpBackend
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := NominatimNlpBackend
+
+nominatim_root  := $(LOCAL_PATH)
+nominatim_out   := $(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
+nominatim_build := $(nominatim_root)/build
+nominatim_apk   := build/outputs/apk/NominatimNlpBackend-release-unsigned.apk
```

Comment:
In this case it would be `NominatimGeocoderBackend-release-unsigned.apk` -> name of the repo again